joomla/joomla-cms
# Added switches to the admin module User Status (mod_status)

Changes:
1. Added switches and make the link parameters work in the Administrator module "mod_status" so the links can be switched on and off.
2. Updated the language file.
3. The second switch is added as extra link to open a new Administrator window for easy Joomla management.

Test instruction:
1. Go to the Module manager in the Administrator and set the top filter to Administrator
2. Open the module "User Status"
3. Turn on and off the "Show view site" and "Show view Administrator" switches and check if both links disappear and reappear in the bottom of the site (See screenshot).

Click the "Show Site" and the "Show Admin" links in the bottom of the Administrator area and check if they open a new Site window and an new Administrator Window. The links should also work when editing and article (See screenshot).

![footer-links-site-admin-joomla](https://cloud.githubusercontent.com/assets/5610413/8149067/ec7947ec-12b7-11e5-8f27-a74fc63980ff.jpg)
